packages/devtools_server/lib/src/handlers.dart

```diff
@@ -83,18 +87,88 @@ class ServerApi {
     switch (request.url.path) {
       case 'api/logScreenView':
         return api.logScreenView(request);
+
+      // ----- Flutter Tool GA store. -----
+      case 'api/getFlutterGAEnabled':
+        // Is Analytics collection enabled?
+        return api.getCompleted(
+          request,
+          '${Usage.doesStoreExits ? _usage.enabled : 'null'}',
+        );
+      case 'api/getFlutterGAClientId':
```

Comment:
how will we get the client id for non flutter users?

Reply:
We won't the clientId will be null and we won't pass it as the userId.